catt/controllers.py

```diff
@@ -164,6 +166,14 @@ class StateFileError(Exception):
     pass
 
 
+class ListenerError(Exception):
```

Comment:
We should probably put these exceptions into their own file, if we have more than two or three.

Reply:
I'll put that on my todo. Should all of our exceptions be in this file, regardless of flavour?

Reply:
Good question, probably not. For example, the ones in `controllers` should be different from the one that inherits from `ClickException` from the CLI. If we only have two here, we should leave them for now, it's just something to keep in the back of our mind for later.